src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationQueryObjectType.cs

```diff
@@ -13,6 +18,27 @@ public LocalizationQueryObjectType(IStringLocalizer<LocalizationQueryObjectType>
 
             Field(x => x.Culture).Description(S["The culture for your content item."]);
             Field(x => x.LocalizationSet).Description(S["The localization set for your content item."]);
+
+            Field<ListGraphType<ContentItemType>, IEnumerable<ContentItem>>()
```

Comment:
this should be probably be `ContentItemInterface`, not `ContentItemType`

Reply:
done. works better in GraphQL now too for casting :)